persistence/src/test/java/com/epam/rft/atsy/persistence/repositories/StatesRepositoryIT.java

```diff
@@ -1,4 +1,4 @@
-package com.epam.rft.atsy.persistence.repositories;
+/*package com.epam.rft.atsy.persistence.repositories;
```

Comment:
Please maintain the test code as well.
The pull request will not be accepted without tests.